### DISCUSSION TITLE
When is the menu supposed to appear?

### DISCUSSION THREAD

**Discussion by geoffreyvanwyk** [alexwforsythe/tmux-which-key#4] [Q&A] [answered]
Is the menu supposed to appear after I enter the `<prefix>` (`Ctrl-B`) key?

At the moment, nothing happens. I am using tmux 3.2a.

**Comment by alexwforsythe** [comment, marked as answer]
With the [default config](https://github.com/alexwforsythe/tmux-which-key/blob/main/config.example.yaml#L5), the menu should appear when you press <kbd>prefix</kbd> + <kbd>Space</kbd>.

If that doesn't work, another tmux plugin might be using the same keybinding, such as tmux-thumbs. You can confirm this by checking your prefix keymap:
```sh
> # My only keybinding for prefix+Space is tmux-which-key's display-menu command
> tmux lsk -Tprefix | grep Space
bind-key    -T prefix Space   display-menu -T "#[align=centre,bold]#[fg=green,align=centre,bold]tmux#[align=centre,bold]" -x R -y P Run space command-prompt "Last window" tab last-window "Last pane" ` last-pane Copy c "show-wk-menu #{@wk_menu_copy}" '' +Windows w "show-wk-menu #{@wk_menu_windows}" +Panes p "show-wk-menu #{@wk_menu_panes}" +Buffers b "show-wk-menu #{@wk_menu_buffers}" +Sessions s "show-wk-menu #{@wk_menu_sessions}" +Client C "show-wk-menu #{@wk_menu_client}" '' Time T clock-mode "Show messages" \~ show-messages +Keys ? "list-keys -N"
```

You can change <kbd>Space</kbd> to another key by editing `keybindings.prefix_table` in `config.yaml`. You can set up a global key binding by uncommenting `keybindings.root_table` (and optionally changing it to a custom key). For example, I use <kbd>Ctrl</kbd>+<kbd>Space</kbd> (`C-Space`) to open the menu without using the prefix key.

If the menu still won't appear, you might be running into [this issue](https://github.com/alexwforsythe/tmux-which-key/tree/main?tab=readme-ov-file#menu-or-submenu-doesnt-appear). Let me know if you still need help!

**Reply by geoffreyvanwyk** [reply]
Thanks @alexwforsythe . It is working now.
